CIP-0048/README.md

```diff
@@ -0,0 +1,249 @@
+      ]
+    }
+    ```
+  - The `image` tag can still be used as a thumbnail if required.
+- If there are multiple `refs` defined the references that are _NOT_ defined in the `image` tag have the same behavior as the `files` tag currently defined in CIP25
+
+# Example
+
+Mint transaction 1
+
+```json
+{
+  "721": {
+    "ext": ["cip48"],
+    "0000000000000000000000000000000000000000000000000000000A": {
+      "NFT0": {
+        "project": "CIP48 Example",
+        "name": "NFT0",
+        // references
+        "refs": [
+          {
+            "name": "NFT0",
+            "mediaType": "text/plain",
+            "src": ["0", "2"]
+          }
+        ]
+      },
+      // payload
+      "p": {
+        "0": ["Hello"],
+        "1": ["Goodbye"]
+      }
+    }
+  }
+}
+```
+
+Mint transaction 2
+
+```json
+{
+  "721": {
+    "ext": ["cip48"],
+    "0000000000000000000000000000000000000000000000000000000A": {
+      "NFT1": {
+        "project": "CIP48 Example",
+        "name": "NFT1",
+        // references
+        "refs": [
+          {
+            "name": "NFT1",
+            "mediaType": "text/plain",
+            "src": ["0", "3"]
+          }
+        ]
+      },
+      // payload
+      "p": {
+        "2": ["World"],
+        "3": ["Moon"]
+      }
+    }
+  }
+}
+```
+
+### Pseudo code walk through using the above example (using default behavior)
+
+1. Find all transactions for the given `<policy_id>`
+
+   - `found 2 mint transcations`
+
+2. Check the **ext** tag for **cip48**
+
+   - `found cip48` we now know the metadata contains references and payloads
+
+3. Iterate over each transaction. If a payload is found append that to an map or some data structure
+
+   - found 4 payloads
+   - ```js
+     payloads = { 0: "Hello", 1: "Goodbye", 2: "World", 3: "Moon" };
+     ```
+
+4. Find all `refs` (references) for the `<asset_name>` we use NFT0. Then build the data in the order defined by the `src` array.
+
+   - found 2 references
+   - ```js
+     nft0_refs = ["0", "2"];
+     nft0_data = "HelloWorld";
+     ```
+
+5. use the mediaType (mimetype) to determine the data.
+   - mediaType = "test/plain"
+   - NFT0 contains text "HelloWorld"
+
+# Backwards compatibility
+
+Handled via the use of the `"ext"` tag defined in CIP-48.
+
+# Further considerations
+
+- Later we may want to use different reference types, maybe even plutus contract references. Due to this I have defined the `type` tag inside the `refs` (references) tag.
+- `p` is used instead of **payload** to reduce bytes. Specifically `p` is not used within the `<asset_name>` tag therefore readability is less of a priority than data size. Unlike `refs` which is defined in the `<asset_name>` tag
+- JSON is kept over CBOR or alternatives as this CIP is encapsulated in the 721 metadatum tag and JSON is easier for developers to adopt
+
+### Duplicate data
+
+There could be issues with duplicate payloads. To solve the payload defined in the most recently minted tx takes priority.
```

Comment:
```suggestion
## Rationale: how does this CIP achieve its goals?

### Further considerations

- Later we may want to use different reference types, maybe even plutus contract references. Due to this I have defined the `type` tag inside the `refs` (references) tag.
- `p` is used instead of **payload** to reduce bytes. Specifically `p` is not used within the `<asset_name>` tag therefore readability is less of a priority than data size. Unlike `refs` which is defined in the `<asset_name>` tag
- JSON is kept over CBOR or alternatives as this CIP is encapsulated in the 721 metadatum tag and JSON is easier for developers to adopt

### Duplicate data

There could be issues with duplicate payloads. To solve the payload defined in the most recently minted tx takes priority.
```
Just sketching out a _Rationale_ from the points already included here... maybe also some of what you already wrote above regarding your design decisions.

Especially I think in the _Rationale_ section you should include:
- why you originally solved this problem with the extra tag, and
- why you abandoned that idea when CIP88 came along.